build/install

```diff
@@ -4,7 +4,4 @@ set -e
 
 echo "  --> Installing last compiled binary (${KRAUT_BUILD_BINARY_PATH}) to: ${KRAUT_INSTALL_PATH}"
 
-# Get rid of existing binary
-rm -f "${KRAUT_INSTALL_BINARY}"
```

Comment:
this line removes a path based on a variable that can be affected by env variables.  This is unnecessary, and so we can mitigate the risk.